deploy/backplane/lpsre/hypershift/management-cluster/10-lpsre-management-cluster-project.ClusterRole.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-lpsre-management-cluster-project
+rules:
+- apiGroups:
+  - package-operator.run
+  resources: ['*']
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+  - create
```

Comment:
I would leave this out if not also granting `delete`.  Else you're stuck in how you can remediate incorrect creations.  My recommendation is use elevated access for both create and delete, these should not be typical for supporting?